packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/surroundingPair/parseTree/python/chuckMatching.yml

```diff
@@ -1,7 +1,7 @@
 languageId: python
 command:
   version: 1
-  spokenForm: change inside
+  spokenForm: change pair
```

Comment:
Weird 😅. Looks right tho

Reply:
Definitely. If you look at the file name. This isn't even the original spoken form.